tests/test-extensions/some-extension/src/main/java/org/mule/test/some/extension/SomeExtension.java

```diff
@@ -7,22 +7,36 @@
 
 package org.mule.test.some.extension;
 
+import org.mule.runtime.extension.api.annotation.Configuration;
 import org.mule.runtime.extension.api.annotation.Export;
 import org.mule.runtime.extension.api.annotation.Extension;
 import org.mule.runtime.extension.api.annotation.Operations;
 import org.mule.runtime.extension.api.annotation.Sources;
 import org.mule.runtime.extension.api.annotation.connectivity.ConnectionProviders;
 import org.mule.runtime.extension.api.annotation.dsl.xml.Xml;
 import org.mule.runtime.extension.api.annotation.error.ErrorTypes;
+import org.mule.sdk.api.annotation.param.Optional;
```

Comment:
For 4.3.0, this annotations will not work, use org.mule.runtime.extension.api.annotation.Optional and org.mule.runtime.extension.api.annotation.Parameter

Reply:
Fixed